actionpack/lib/action_dispatch/journey/scanner.rb

```diff
@@ -7,64 +7,62 @@
 module ActionDispatch
   module Journey # :nodoc:
     class Scanner # :nodoc:
+      STATIC_TOKENS = Array.new(150)
```

Comment:
Doesn't really matter in the slightest, but I think 128 would make more sense here (or 125 if you want to get pedantic).